ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -231,102 +232,121 @@ Item {
             }
         }
 
-        StatusChatInput {
-            id: chatInput
-
+        Item {
```

Comment:
I just added a `RowLayout` here https://github.com/status-im/status-desktop/pull/11073
With your changes it will already be 3 items inside, I propose keeping the layout comparing to anchors, wdyt?

Reply:
As this panel needs to be shown on top of the chat input it can't just be a part of the row layout, nevertheless I've rebased and adapted.

Reply:
hmm, couldn't `ChatPermissionQualificationPanel` be child of `StatusChatInput`? 🤔 
This would make it unnecessary to create another level of nesting with `Item`.

```qml
RowLayout {
    StatusChatInput {
        // set properties
        ChatPermissionQualificationPanel {
            // width/height/anchors whatever
        }
    }
    Button {
        text: "Unlock"
    }
}
```

Reply:
nope, is it is then then when then chat input is disabled (because this panel is visible), it's not possible to open the popup or scroll (since it's disabled too due to its parent being disabled)